docs/DeveloperGuide.md

```diff
@@ -1277,6 +1277,49 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Marking tasks as done (and undone)
+
+1. **Marking a task as done:**
+
+    1. Pre-requisites:
+       - You have at least one undone task (see `add task`).
+       - You are not currently `cd`'d into a module (`cd ..` if you are).
+       - All tasks are being shown (i.e. run `list task -a` first). 
+    1. Test case: `done task 1`<br>
+       Expected: The first task (topmost in the UI) is marked as done (with a "done" label) and a success message is displayed.
+    1. Test case: `done task 1` followed by another `done task 1` <br>
+      Expected: An error message saying that the task is already marked as done is displayed.
```

Comment:
Should we stick to exactly one task? Since if a user has >1 undone tasks, `done task 1` will send that task to the end of the list, and the second `done task 1` will go through on the previously #2 undone task